tools/protodoc/protodoc.py

```diff
@@ -133,6 +133,10 @@
     'breaking changes. Do not use this feature without understanding each of the previous '
     'points.\n\n')
 
+FILE_LEVEL_NOT_IMPLEMENTED = (
```

Comment:
im wondering whether we want a message, or rather no page at all

Reply:
My initial idea was to completely hide the document. But the protoc plugin always need a output. orz

Reply:
ah, i see the comments above now

wondering if there is a way around this as it seems messy to include empty/hidden pages

Reply:
what about hacking `api_proto_plugin/plugin.py` to bail if its an empty proto file ?

Reply:
I have tried this approach. I skip the output file generating or return output file with empty content. But I still failed. Perhaps because I am not familiar with this tool yet, there are other better methods. But I cannot be sure at the moment.

Reply:
there is already a situation in which the proto plugin skips certain files - see here:

https://github.com/envoyproxy/envoy/blob/8fc53a00ebc085024d88fddd93e9051413355a0b/tools/api_proto_plugin/plugin.py#L69-L70

i may be wrong, but i think we can do something similar for hidden proto files

Reply:
@phlax it not works.

If it's independent API (not link in any doc tree):
```
/tmp/tmpqc31del1/generated/rst/api-v3/service/endpoint/v3/leds.proto.rst: WARNING: document isn't included in any toctree
```
If it's normal API:
```
/tmp/tmpqc31del1/generated/rst/api-v3/config/filter/network/network.rst:4: WARNING: toctree contains reference to document 'api-v3/extensions/filters/network/echo/v3/echo.proto' that doesn't have a title: no link will be generated
```